includes/managers/elements.php

```diff
@@ -298,6 +298,13 @@ private function init_categories() {
 			],
 		];
 
+		$this->categories = array_merge_recursive( [
```

Comment:
Why add it like this and not directly to the array?

Reply:
@udidol Updated.